maturity-model.md

```diff
@@ -9,10 +9,10 @@ Tier | Content
 0 | IDC Minimalization > Project Formation using Socio-Economic Lens & #DifferentTogether Pledge

```

Comment:
How about using 'and' instead of '&'?  Beside a '#' it looks like a typo.

Reply:
That makes sense, I'll change that.